### PR TITLE
# perk-store

### PR DESCRIPTION
**Pull request by Rdidmohamedelhadi**

### CODE REVIEW

**Review by GouadriaHanen — DISMISSED** [review]

File: perk-store-services/src/main/java/org/exoplatform/perkstore/rest/PerkStoreProductREST.java [hunk 1]
```diff
@@ -16,28 +16,25 @@
...
-import io.swagger.annotations.*;
+import javax.annotation.security.RolesAllowed;
+import javax.ws.rs.*;
+import javax.ws.rs.core.*;
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.util.List;
+
+import static org.exoplatform.perkstore.service.utils.Utils.computeErrorResponse;
+import static org.exoplatform.perkstore.service.utils.Utils.getCurrentUserId;
```
Comment (boubaker) [inline perk-store-services/src/main/java/org/exoplatform/perkstore/rest/PerkStoreProductREST.java]: static imports should be at first, why changing the imports ?

File: perk-store-services/src/main/resources/locale/addon/PerkStore_az.properties [hunk 2]
```diff
@@ -237,7 +238,7 @@ exoplatform.perkstore.button.refresh=Refresh
 exoplatform.perkstore.button.search=Search
 exoplatform.perkstore.button.download=Download
 exoplatform.perkstore.button.loadMore=Load more
-exoplatform.perkstore.button.editProduct=Edit product
+exoplatform.perkstore.button.editProduct=Edit Product
```
Comment (boubaker) [inline perk-store-services/src/main/resources/locale/addon/PerkStore_az.properties:241]: changing keys will not be impacted and will be overwritten on next Crowdin synchro, please read :
http://developer.exoplatform.org/#id-how-to-update-an-existing-translation-string

File: perk-store-webapps/src/main/webapp/vue-app/components/PerkStoreApp.vue [hunk 3]
```diff
@@ -360,17 +278,23 @@ import ProductNotification from './perk-store/ProductNotification.vue';
 import {initSettings, getOrderFilter, getProductFilter, storeProductFilter} from '../js/PerkStoreSettings.js';
 import {toFixed} from '../js/PerkStoreProductOrder.js';
 import {getProductList, getProduct} from '../js/PerkStoreProduct.js';
+import CreateProductButton from './perk-store/CreateProductButton.vue';
```
Comment (boubaker) [inline perk-store-webapps/src/main/webapp/vue-app/components/PerkStoreApp.vue]: This is a bad practice to declare an import inside a vue file.
As made in other addons, you should update this module by defining initComponents.js where you can import and declare the Vue files.

Comment (Rdidmohamedelhadi) [reply perk-store-webapps/src/main/webapp/vue-app/components/PerkStoreApp.vue]: done !

File: perk-store-services/src/main/resources/locale/addon/PerkStore_az.properties (hunk 2 above)
Comment (Rdidmohamedelhadi) [reply perk-store-services/src/main/resources/locale/addon/PerkStore_az.properties:241]: fixed in crowdin !!

**Review by GouadriaHanen — DISMISSED** [review]

**Review by GouadriaHanen — APPROVED** [review]

**Pull request merged**